xray/test/test_plot.py

```diff
@@ -284,6 +404,13 @@ def test_extend(self):
         artist = self.plotmethod(vmin=-10, vmax=0)
         self.assertEqual(artist.extend, 'max')
 
+    def test_levels(self):
```

Comment:
I'm fairly sure it was passed through as a `kwarg`. I added it to the signature so its there now.